packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -51,6 +52,7 @@ export class LocalChannelContext implements IChannelContext {
         dirtyFn: (address: string) => void,
         private readonly snapshotTree: ISnapshotTree | undefined,
     ) {
+        this.services = new Lazy(() => this.servicesGenerator());
```

Comment:
>servicesGenerator [](start = 44, length = 17)

i think i would prefer if this was just inclined, as the function should only be called once and here

Reply:
done